assets/src/application/ui/layout/entityList/EntityTable.tsx

```diff
@@ -41,4 +40,4 @@ const EntityTable = <E extends Entity, FS extends ELFSM>({
 	);
 };
 
-export default React.memo(EntityTable, entitiesUnchanged);
+export default EntityTable;
```

Comment:
This memoization didn't allow `displayStartOrEndDate` to show effect, so I removed it.